contributors/devel/release/releasenotes.md

```diff
@@ -0,0 +1,94 @@
+# Creating PRs for release notes
+
+This document describes how to create PRs in Kubernetes repos so that the changes can be tracked with the
+Kubernetes release and published in release notes.
+
+## Curated release notes
+
+The release notes for each Kubernetes release are divided into several sections.
+
+All sections are grouped into subsections by component.
+
+#### Action required
+
+The action required section outlines any steps the user must perform before upgrading their cluster.
+The may include actions such as migrating APIs or commands to new versions, upgrading /
+installing tools, or validating the cluster environment.
+
+#### Deprecations
+
+The deprecation section outlines any APIs, tools, commands or flags that are being deprecated.
+They will continue to be available for this release, but will be removed in a subsequent release.
+
+#### Feature highlights
+
+The feature highlights section is manually populated from the [features repo] and contains
+a list of the feature highlights for the release. 
+
+#### Notable changes
+
+The notable changes section is generated from the PR release notes, and contains a list
+of features and bug fixes going into the release.
+
+## Release note labels
+
+All  PRs with user visible changes must have a release note attached.  This will automatically
+
+populate them in the release notes.
+
+Release notes are added applying on of the `release-note*` labels by adding a section
```

Comment:
These labels should be applicable via community members who do not have labeling privileges on the repo in question, eg: via a [bot command](https://github.com/kubernetes/test-infra/blob/master/commands.md) in the comments

(I'd to explicitly avoid spelling out the syntax here lest it change in the future)

Reminder that new labels like `-deprecation` should be added to the commands if they're not presently supported

Reminder that the labels should be consistently applied to other repos in the kubernetes org beyond kubernetes/kubernetes for this to take effect